app/src/main/java/com/hedvig/app/feature/settings/SettingsActivity.kt

```diff
@@ -70,47 +70,33 @@ class SettingsActivity : BaseActivity() {
                 }
             }
 
-            val marketPreference = findPreference<ListPreference>(SETTINGS_MARKET)
-            if (market != null) {
-                marketPreference?.setValueIndex(market.ordinal)
-            } else {
-                MarketPickerActivity.newInstance(requireContext())
+            val marketPreference = findPreference<Preference>(SETTINGS_MARKET)
+            if (market == null) {
+                MarketingActivity.newInstance(requireContext())
```

Comment:
What's going on here?

Reply:
I removed the picker so you just press `Market` and dont select the market in the settings activity. I remember we talked about this before. But now i realize that i should have talked to Zak about this before changing

Reply:
Yes, but does this code actually do that? 😉 It looks to me like it only creates an `Intent` to start the `MarketingActivity` but never actually starts it 🙂

Reply:
Ahhh, didnt even notice that 😅